src/platform/forms-system/src/js/fields/ObjectField.jsx

```diff
@@ -201,6 +201,10 @@ class ObjectField extends React.Component {
       'schemaform-block': title && !isRoot,
     });
 
+    const newFieldsetClassNames = classNames(fieldsetClassNames, {
```

Comment:
Could we construct this all at once on line 183 and just call it fieldsetClassNames?

Reply:
We definitely can.